tests/model_serving/model_server/private_endpoint/conftest.py

```diff
@@ -0,0 +1,207 @@
+import json
+import pytest
+from typing import Generator
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.secret import Secret
+from ocp_resources.namespace import Namespace
+from ocp_resources.serving_runtime import ServingRuntime
+from ocp_resources.pod import Pod
+from simple_logger.logger import get_logger
+from ocp_resources.service_mesh_member import ServiceMeshMember
+from kubernetes.dynamic import DynamicClient
+
+from tests.model_serving.model_server.private_endpoint.utils import create_sidecar_pod, get_flan_pod, b64_encoded_string
+from tests.model_serving.model_server.private_endpoint.infra import create_ns
+from tests.model_serving.model_server.private_endpoint.constants import (
+    AWS_REGION,
+    AWS_BUCKET,
+    AWS_ENDPOINT,
+    SR_ANNOTATIONS,
+    SR_CONTAINERS_KSERVE_CAIKIT,
+    SR_SUPPORTED_FORMATS_CAIKIT,
+    SR_VOLUMES,
+)
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="module")
+def endpoint_namespace(admin_client: DynamicClient) -> Generator[Namespace]:
+    yield from create_ns(admin_client=admin_client, name="endpoint-namespace")
+
+
+@pytest.fixture(scope="module")
+def diff_namespace(admin_client: DynamicClient) -> Generator[Namespace]:
+    yield from create_ns(admin_client=admin_client, name="diff-namespace")
+
+
+@pytest.fixture()
+def endpoint_sr(
+    admin_client: DynamicClient,
+    endpoint_namespace: Namespace,
+) -> Generator[ServingRuntime]:
+    with ServingRuntime(
```

Comment:
why not use `caikit-tgis-serving-template`  template?
see https://github.com/opendatahub-io/opendatahub-tests/pull/21

Reply:
I was not aware of the possibility, see https://github.com/opendatahub-io/opendatahub-tests/pull/22#discussion_r1815032259